### DISCUSSION TITLE
How to not get encoding in the title of a page.

### DISCUSSION THREAD

**Discussion by lizneely** [thegetty/quire#167] [Get Help] [answered]
Hi everyone! 

I'm new to Quire and am putting together our first small catalogue to accompany an exhibition at the Georgia O'Keeffe Museum. Please forgive me if this is a silly question. I'm not a programmer but still try to make cool things!

On the catalogue pages, I am using (I also tried it without the quotes):

```
---
title: "O'Keeffe's Dishes"
type: entry
weight: 101
class: side-by-side
object:
  - id: 1
---
```

But this means that the title of the page in preview and on the static site (for SEO, bookmarking, etc.) comes up as:

**`O&rsquo;Keeffe&rsquo;s Dishes | Josephine Halvorson`**

As you can imagine, I have a lot of titles with O'Keeffe and will certainly continue to in future catalogues.

Is there an easy way that I can get the proper title on the page:

**O'Keeffe's Dishes | Josephine Halvorson**

I'm loving the platform and having fun putting this together. 

Thanks,
Liz

Liz Neely
Curator of Digital Experience
Georgia O'Keeffe Museum
Santa Fe, N.M.

**Comment by geealbers** [comment, marked as answer]
Hi @lizneely, thanks for posting about this. @swambold1 ran into this issue recently as well. The immediate solution is to use [unicode right single quotation marks](https://unicode-table.com/en/2019/) in place of the apostrophes:

```yaml
title: "O’Keeffe’s Dishes"
```

(These can be typed on a mac keyboard with `Shift-Option-]`.)

Long-term, I'll push a fix for this in future versions of Quire. The problem arises because the page title is being processed Quire's markdown processor which also converts straight apostrophes to curly right single quotation marks. For the `<title>` element through, Quire takes the additional step of stripping out any HTML from the title but is being a little greedy about is and ends up messing up the HTML entity for the single quotation marks as well. The fix I'm going to push will remove all processing altogether so that the title in the page and the browser tab just ends up as it's typed. Regardless of apostrophes or right single quotes. This will also mean that titles with italics will show up as something like `O’Keeffe’s *White Abstraction*`.

**Reply by lizneely** [reply]
Thank you, @geealbers! -- that fix works for me. I figured it was in the processing, but I'm happy there is a simple way around it until a longer-term fix recognizes the page title as a special field. 

(By the way, preview rocks!)
